cosmic_ray/operators/exception_replacer.py

```diff
@@ -0,0 +1,20 @@
+import ast
+
+from .operator import Operator
+
+
+class ExceptionReplacer(Operator):
+
+    """An operator that modifies exception handlers."""
+
+    def visit_ExceptHandler(self, node):  # noqa
+        return self.visit_mutation_site(node)
+
+    def mutate(self, node, _):
+        """Modify the exception handler with another exception type."""
+        except_id = 'TypeError' if node.name.id == 'SystemError' else \
```

Comment:
What if they have a previous handler for the exception that you mutate to? That seems like a case of a truly undetectable equivalent mutation.

I can think of two ways to try to address this. One would be to traverse the other handlers and pick an exception that isn't handled. This could get messy. Another is to insert our own, "private" exception type into the module namespace and then handle that. Offhand, I like this approach a bit more.

As written, your code is probably just fine for almost all users. Maybe the thing to do is to open a new issue that gives us a place to think about this.